examples/projections/nongeo/polar.py

```diff
@@ -61,7 +61,7 @@
     # Set map width to 5 cm
     projection="P5c",
     # Set the frame and title; @^ allows for a line break within the title
-    frame=["xa45f", "+gbisque+tprojection='P5c' @^ region=[0, 360, 0, 1]"],
+    frame=["xa45f", '+gbisque+tprojection="P5c" @^ region=[0, 360, 0, 1]'],
```

Comment:
Hmm. I am a bit debating on what is more important, (i) using consistently double quotation marks for string arguments or (ii) making the title consistent with the argument. Currently, I am a bit more on the side of (i). A way to use both times double quotation marks is to add a leading `\` for the inner double quotation marks. But maybe this makes things only more complicated.

```suggestion
    frame=["xa45f", "+gbisque+tprojection=\"P5c\" @^ region=[0, 360, 0, 1]"],
```

Reply:
Backslashes make the codes more difficult to read and should be avoided. 

Personally, I'm OK with displaying either single or double quotes in the plot title. Currently, single quotes are displayed like `’`, rather than `'`. See https://github.com/GenericMappingTools/pygmt/issues/3476 for the reason and #3105 for the fix.

Reply:
> Currently, single quotes are displayed like `’`, rather than `'`. See #3476 for the reason and #3105 for the fix.

That's the reason why I wanted to change it. The single quotes are looking like accute accents right now and that's something that seems to be weird for users when they see it in plots.

Reply:
I would still go with using double quotations here since the titles directly reference the `projection` parameter in the code. Using single quotations for a string in such cases is fine in my opinion. But I'm also fine if all others what to keep it as is.

Reply:
I'm inclined to keep it as is.

Reply:
Would also prefer to keep it as is, as I prefer to have (if possible) double quotation marks for the string inputs.